index.js

```diff
@@ -0,0 +1,45 @@
+
+
+function randomNumber() {
+    
+    var numberlist = [1,2,3,4,5,6,7,8,9];
+    var picknumber = [];
+    for (var i = 0; i < 3; i++){
+        var tnum = [];
+        tnum = numberlist.splice(Math.floor(Math.random() * (9 - i)), 1)[0];
+        picknumber.push(tnum);
+    }
+    console.log(picknumber);
+}
+
+document.getElementById("start").addEventListener('click', function() {
+    randomNumber();
+});
+
+
+//값 입력
+function inGame() {
+
+var strike = 0;
+var ball = 0;
+var answer = document.getElementById("answer");
+
+
+for(var n = 0; n < 3; n++){
+    if(answer[n] === randomNumber[n])
```

Comment:
randomNumber는 함수인데 배열에서의 데이터 접근 방식으로 작성되었네요! for문 안에서 randomNumber가 실행되면 결과값이 계속 달라지기 때문에 randomNumber 함수의 결과 값을 변수에 할당 한 뒤 비교 할 수 있을 것 같습니다.
```suggestion
  let targetNumbers = randomNumber();
  if (answer[n] === targetNumber[n]) {
```